### ISSUE TITLE
Add a page for WASM/other language proposals?

### ISSUE THREAD

**Issue by saadq** [saadq/proposals.es#11]
https://github.com/WebAssembly/proposals

**Comment by saadq** [comment]
I've started work on this. I want to change the current project structure to be a monorepo with these packages:

* `ecma` (current app)
* `wasm` (new app)
* `common` (common components/utils to share between the two apps)

I'm looking into using the [Multi Zones](https://nextjs.org/docs/advanced-features/multi-zones) feature for Next.js to accomplish this, hoping to be able to keep https://proposals.es pointing to the current app and then have https://wasm.proposals.es point to the WASM app.

**Comment by saadq** [comment]
Decided against using multi zones and just kept it as 2 separate apps.

* https://proposals.es => ECMAScript proposals
* https://wasm.proposals.es => WASM proposals

The separate app/monorepo is setup now so we can get started on moving a bunch of stuff from `ecma` to the `common` folder and then reusing it in the `wasm` app.

**Comment by ljharb** [comment]
Doesn’t the “es” mean “ecmascript”? WASM isn’t part of ecma.

**Comment by saadq** [comment]
Yep, you're absolutely right :p. I figured it was still fine to reuse the domain, but now that you've called that out it feels pretty dirty to use it lol since it might cause confusion to people.

@ljharb curious if you have any opinions on what a good domain would be for it? :) 

I was considering using `wasmproposals.*` which has several options like:

* wasmproposals.com
* wasmproposals.dev
* wasmproposals.io

There's also `wasmproposa.ls` and `proposalswa.sm`

**Comment by saadq** [comment]
Ok, I think problem solved... I've bought `proposals.dev`.

So I think I'll have something like `ecma.proposals.dev` and `wasm.proposals.dev`, and I'll keep `proposals.es` and `ecma.proposals.dev` linked.

It will let me also make it easier to have a home page so users can navigate to either WASM/ECMA

**Comment by saadq** [comment]
Languages that have proposals which can be tracked:

* WASM
* Swift
* Golang
* Rust
* Python (will prob be harder to parse Python proposals tho)
